dist/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To do list</title>
+
+    <link rel="stylesheet" href="https://bootswatch.com/5/sketchy/bootstrap.min.css"/>
+<script defer src="/runtime.js"></script><script defer src="/main.js"></script></head>
+<body>
+    <h1>TO-DO List</h1>
+    <div class="tasks-wrapper">
+      <div class="tasks-title"><label>Today's To Do</label><i class="fa-solid fa-arrows-rotate refresh-icon"></i></div>
+      <form class="todo-list">
+        <input class="input-text" type="text" id="input-task" placeholder="Add to your list...">
+        <button class="submit-btn" type="submit">Add</button>
+      </form>
+      <div class="todo-list-items"></div>
+
+      <p class="clear-all">Clear all completed</p>
```

Comment:
Hi, I suggest you wrap this section in a section tag to ensure a cleaner and well-separated code.